portal-ui/src/screens/Console/Tenants/securityContextSelector.tsx

```diff
@@ -0,0 +1,132 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Fragment } from "react";
+import InputBoxWrapper from "../Common/FormComponents/InputBoxWrapper/InputBoxWrapper";
+import FormSwitchWrapper from "../Common/FormComponents/FormSwitchWrapper/FormSwitchWrapper";
+import { Grid } from "@mui/material";
+import { useDispatch } from "react-redux";
+import createStyles from "@mui/styles/createStyles";
+import withStyles from "@mui/styles/withStyles";
+
+interface IEditSecurityContextProps {
+    classes: any;
+    runAsUser: string;
+    runAsGroup: string;
+    fsGroup: string;
+    runAsNonRoot: boolean;
+    setRunAsUser: any;
+    setRunAsGroup: any;
+    setFSGroup: any;
+    setRunAsNonRoot: any;
+}
+
+const styles = () =>
+  createStyles({
+    tabsContainer: {
```

Comment:
```suggestion
    tabsContainer: {
      display: "flex",
      height: "100%",
      width: "100%",
      gap: "8px"
    }
```

`gap` property could be added to give spacing horizontally and vertically in responsive mode.
e.g:
![image](https://user-images.githubusercontent.com/23444145/175477011-ec07536c-a429-4c7a-971d-48550ffd756e.png)

![image](https://user-images.githubusercontent.com/23444145/175477198-31a0c820-c808-4307-a1b1-491710281325.png)